### PR TITLE
Unify how we refer to involved people in report events

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

**Comment by hennevogel** [comment]
Some day, this should probably be followed up by https://trello.com/c/hqkiRR1d

**Comment by danidoni** [comment]
> Some day, this should probably be followed up by https://trello.com/c/hqkiRR1d

Yes! Please!

File: src/api/app/models/event/report_for_user.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 module Event
   class ReportForUser < Report
     self.description = 'Report for a user created'
-    payload_keys :user_login
+    payload_keys :accused
```
Comment (danidoni) [inline src/api/app/models/event/report_for_user.rb:4]: What about `offender`?

Comment (hennevogel) [reply src/api/app/models/event/report_for_user.rb:4]: I'm not a native speaker but I guess you are only an offender if the accusations against you are found to be true...

Comment (hennevogel) [reply src/api/app/models/event/report_for_user.rb:4]: @srbaker ?

Comment (danidoni) [reply src/api/app/models/event/report_for_user.rb:4]: > I'm not a native speaker but I guess you are only an offender if the accusations against you are found to be true...

Hm, didn't know that.

**Review by danidoni — APPROVED** [review]

**Pull request merged**